src/relayFeeCalculator/relayFeeCalculator.ts

```diff
@@ -277,10 +278,10 @@ export class RelayFeeCalculator {
         isDefined(details.addresses[destinationChainId])
     );
     const outputToken = deposit.outputToken.isZeroAddress()
-      ? destinationChainTokenDetails!.addresses[destinationChainId]
-      : deposit.outputToken.toNative();
+      ? toAddressType(destinationChainTokenDetails!.addresses[destinationChainId], destinationChainId)
+      : deposit.outputToken;
```

Comment:
does the zero address need to be converted into the correct base58/bytes32 representation?